sdk/lib/rtl/amd64/unwind.c

```diff
@@ -649,6 +649,18 @@ RtlVirtualUnwind(
     return NULL;
 }
 
+static __inline
+BOOL
+RtlpIsStackPointerValid(
+    _In_ ULONG64 StackPointer,
+    _In_ ULONG64 LowLimit,
+    _In_ ULONG64 HighLimit)
+{
+    return (StackPointer >= LowLimit) &&
+           (StackPointer < HighLimit) &&
+           ((StackPointer & 7) == 0);
```

Comment:
What is that magic 7?

Reply:
Mask for the three lower bits of a value.